### PR TITLE
Adding CyberProof connector Webhook trigger support and semantic changes

### PR DESCRIPTION
**Pull request by DavidMeu**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/CyberProof/apiDefinition.swagger.json [hunk 1]
```diff
@@ -206,6 +206,149 @@
...
+    },
+    "/api/v1/webhooks/azure-logic-apps": {
+      "get": {
+        "description": "CyberProof Triggers List.",
+        "summary": "CyberProof Triggers List",
+        "operationId": "CPGetTriggers",
+        "responses": {
+          "200": {
+            "description": "List of triggers",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/CyberProof/apiDefinition.swagger.json:218]: @DavidMeu Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

File: certified-connectors/CyberProof/apiDefinition.swagger.json [hunk 2]
```diff
@@ -206,6 +206,149 @@
...
+            "in": "path",
+            "description": "webhook id.",
+            "required": true,
+            "x-ms-summary": "webhook id"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Trigger deregistration requested",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/CyberProof/apiDefinition.swagger.json:250]: @DavidMeu Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/CyberProof/apiDefinition.swagger.json (hunk 1 above)
Comment (DavidMeu) [reply certified-connectors/CyberProof/apiDefinition.swagger.json:218]: @sriyen-msft But this is internal operation

File: certified-connectors/CyberProof/apiDefinition.swagger.json (hunk 2 above)
Comment (DavidMeu) [reply certified-connectors/CyberProof/apiDefinition.swagger.json:250]: @sriyen-msft But this is internal operation

File: certified-connectors/CyberProof/apiDefinition.swagger.json [hunk 3]
```diff
@@ -206,6 +206,149 @@
...
+        "description": "CyberProof Triggers List.",
+        "summary": "CyberProof Triggers List",
+        "operationId": "CPGetTriggers",
+        "responses": {
+          "200": {
+            "description": "List of triggers",
+            "schema": {}
+          }
+        },
+        "x-ms-visibility": "internal"
```
Comment (DavidMeu) [inline certified-connectors/CyberProof/apiDefinition.swagger.json:221]: @sriyen-msft Also see x-ms-visibility property marked as internal

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**